trunk/SUAVE/Methods/Aerodynamics/Airfoil_Panel_Method/airfoil_analysis.py

```diff
@@ -0,0 +1,219 @@
+    del_star_tr_top = del_star_t_top[i_tr_top] 
+    theta_tr_top    = theta_t_top[i_tr_top]    
+    delta_tr_top    = delta_t_top[i_tr_top] 
+    
+    x_h_top, theta_h_top, del_star_h_top, H_h_top, cf_h_top, delta_h_top = heads_method(delta_tr_top,theta_tr_top, del_star_tr_top, L_top - x_tr_top, Re_L, x_top - x_tr_top, Ve_top, dVe_top)
+    x_ts            = np.concatenate([x_t_top[:i_tr_top], x_h_top + x_tr_top])
+    theta_top       = np.concatenate([theta_t_top[:i_tr_top] ,theta_h_top])
+    del_star_top    = np.concatenate([del_star_t_top[:i_tr_top], del_star_h_top])
+    H_top           = np.concatenate([H_t_top[:i_tr_top] ,H_h_top])
+    cf_top          = np.concatenate([cf_t_top[:i_tr_top], cf_h_top])
+    delta_top       = np.concatenate([delta_t_top[:i_tr_top], delta_h_top]) 
+    
+    # ---------------------------------------------------------------------
+    # Concatenate Upper and Lower Surface Data 
+    # ---------------------------------------------------------------------   
+
+    x_vals                = x[::-1]  
+    y_vals                = y[::-1]  
+    Cp_vals               = np.concatenate([Cp_top[::-1],Cp_bot])
+    Ve_vals               = np.concatenate([Ve_top[::-1],Ve_bot])
+    dVe_vals              = np.concatenate([dVe_top[::-1],dVe_bot ])
+                          
+    theta_top_func        = interp1d(x_ts[::-1], theta_top[::-1])
+    theta_bot_func        = interp1d(x_bs,theta_bot) 
+    delta_star_top_func   = interp1d(x_ts[::-1] ,del_star_top[::-1])
+    delta_star_bot_func   = interp1d(x_bs,del_star_bot) 
+    H_top_func            = interp1d(x_ts[::-1],H_top[::-1])
+    H_bot_func            = interp1d(x_bs,H_bot)   
+    Cf_top_func           = interp1d(x_ts[::-1],cf_top[::-1])
+    Cf_bot_func           = interp1d(x_bs,cf_bot) 
+    delta_top_func        = interp1d(x_ts[::-1],delta_top[::-1])
+    delta_bot_func        = interp1d(x_bs,delta_bot)     
+                          
+    theta_vals            = np.concatenate([theta_top_func(x_top[::-1]),theta_bot_func(x_bot)])
+    delta_star_vals       = np.concatenate([delta_star_top_func(x_top[::-1]),delta_star_bot_func(x_bot)])
+    H_vals                = np.concatenate([H_top_func(x_top[::-1]),H_bot_func(x_bot)])
+    Cf_vals               = np.concatenate([Cf_top_func(x_top[::-1]),Cf_bot_func(x_bot)])
+    delta_vals            = np.concatenate([delta_top_func(x_top[::-1]),delta_bot_func(x_bot)])
+    
+    tr_crit_top_func      = interp1d( x_t_top[::-1], tr_crit_top[::-1]) 
+    tr_crit_bot_func      = interp1d( x_t_bot, tr_crit_bot) 
+    Re_theta_t_top_func   = interp1d(x_t_top[::-1],  Re_theta_t_top[::-1])
+    Re_theta_t_bot_func   = interp1d(x_t_bot,Re_theta_t_bot)
+    Re_theta_t_vals       = np.concatenate([Re_theta_t_top_func(x_top[::-1]), Re_theta_t_bot_func(x_bot)])
+    tr_crit_vals          = np.concatenate([tr_crit_top_func(x_top[::-1]),tr_crit_bot_func(x_bot)])
+     
+    s_vals                = np.zeros_like(x_vals)
```

Comment:
wait so all of these values are just zero and not updated?

Reply:
I removed these place holders we talked about on slack